core/services/run_manager.go

```diff
@@ -204,12 +204,18 @@ SELECT coalesce(max(task_runs.minimum_confirmations), 0) FROM job_runs
 	if maxConfs != 0 {
 		for i := range run.TaskRuns {
 			if run.TaskRuns[i].TaskSpec.Type == adapters.TaskTypeRandom && run.TaskRuns[i].MinRequiredIncomingConfirmations.Valid {
+				// Sanity cap must be less than 256 otherwise the blockhash will not be available directly
+				// We give it 56 blocks as a buffer to still be able to fulfill within that bound.
+				newConfs := maxConfs * 2
+				if maxConfs*2 > 200 {
```

Comment:
minor but you could do `if newConfs > 200 {`

Reply:
Why is it times by two twice... so maxConfs must be less than 50?

Reply:
no just multiplies once - updated to make it more clear with piotrs suggestion